chrome_extension/vendor/mooltipass/device.js

```diff
@@ -329,6 +335,22 @@ mooltipass.device.sendCredentialRequestMessageFromQueue = function()
                     }
                 }
                 if (background_debug_msg > 3) mpDebug.log("%c Emulation mode: nothing in buffer!", mpDebug.css('00ff00'));
+                
+                if (!credentialsFound) {
```

Comment:
wait... if we actually find the credentials, don't we return from the function? (line 328/334)
your !credentialsFound is therefore not necessary, right?